configs/networks.json

```diff
@@ -10,5 +10,20 @@
     "url": "starknetid-sepolia.starknet.a5a.ch",
     "indexer": true,
     "description": "The Starknet testnet"
+  },
+
+  {
+    "name": "stellar-mainnet",
+    "url": "stellar-mainnet-url",
+    "indexer": true,
```

Comment:
Set indexer to false for stellar because we currently don't have one for these networks